layout/static-content/static-content-component.js

```diff
@@ -3,19 +3,24 @@ import PropTypes from 'prop-types';
 import renderHTML from 'react-render-html';
 
 
-const StaticContentComponent = ({ content }) => (
-  <section className="l-content">
-    <article className="l-content-body">
-      <div className="l-container">
-        <div className="row align-center">
-          <div className="column small-12 medium-8">
-            { renderHTML(content) }
+const StaticContentComponent = ({ content }) => {
+  const emptyContent = '<p><br></p>';
```

Comment:
@pablopareja Quill matches visual content even when wysiwyg field is empty with `<p><br></p>`. I can't see a straightforward way to prevent that on an empty field (only stopping completely, which I don't know is desirable, see: https://quilljs.com/docs/modules/clipboard/).

This check only renders the module if there is content from the back office (removing the additional spacing as requested), though perhaps there is a more elegant way?